photo-blocks/functions.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+if ( ! function_exists( 'photo_blocks_support' ) ) :
+	function photo_blocks_support()  {
+
+		// Adding support for featured images.
+		add_theme_support( 'post-thumbnails' );
+
+		// Adding support for alignwide and alignfull classes in the block editor.
+		add_theme_support( 'align-wide' );
+
+		// Adding support for core block visual styles.
+		add_theme_support( 'wp-block-styles' );
```

Comment:
I assume we need these here rather than putting them in theme.json?

Reply:
For now, yes.